lib/i18n/english/tags/blog-composer.js

```diff
@@ -0,0 +1,23 @@
+export default {
+      "titlePlaceholder":"Enter your post title here...",
+      "saveLinkText" : "Save",
+      "savedLinkText" : "Saved",
+      "publishLinkText" : "Publish",
+      "unpublishLinkText" : "Unpublish",
+      "loginButtonText" : "Login to start a post",
+      "dummyUserText" : "Unnamed",
+      "dummyContentText" : "Dummy content...",
+      "linkPromptText":"Enter the link URL",
+      "imagePromptText":"Enter the image URL",
+      "editProgessText" : "Saving...",
```

Comment:
"editProgessText" : "Saving..."
"editErrorText":"Couldn't be saved!"

These are wrong. This values have nothing to do with Edit option. They should be renamed as "saveProgessText", "saveErrorText"